sycl/test-e2e/Matrix/joint_matrix_half_impl.hpp

```diff
@@ -122,6 +122,22 @@ int main() {
 
     if (combinations[i].nsize == 16) { // architecture::intel_gpu_pvc
       test<float, half, 2, /*TM*/ 8, /*TN*/ 16, /*TK*/ 16>();
+      // test<half, half, 2, /*TM*/ 8, /*TN*/ 16, /*TK*/ 16>();
+
+      // This combination is not currently supported for sub group size = 32 in
+      // IGC
+#if (!defined(SG_SZ) || SG_SZ != 32)
+      // test<half, float, /*TM*/ 16, /*TN*/ 16, /*TK*/ 16>();
```

Comment:
Here and below, I think these to parameters should be swapped:
```suggestion
      // test<float, half, /*TM*/ 16, /*TN*/ 16, /*TK*/ 16>();
```